include/erlcloud_aws.hrl

```diff
@@ -37,6 +37,8 @@
           s3_bucket_access_method=vhost::vhost|path|auto,
           s3_bucket_after_host=false::boolean(),
           sdb_host="sdb.amazonaws.com"::string(),
+          cognito_host="cognito-idp.eu-west-1.amazonaws.com"::string(),
```

Comment:
please rename `cognito_host` to `cognito_idp_host`.
Same for the scheme below

Reply:
@kkuzmin  done